PayPalCheckoutWeb/src/androidTest/java/com/paypal/android/checkoutweb/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.paypal.android.checkoutweb
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```

Comment:
Now that we have a model of how we are going to do integration tests for this SDK, should we add integration tests to this module so that it is release ready? Or can we create a JIRA to add them?

Reply:
@sarahkoop i vote for a diferent JIRA ticket, as this PR is blocking some features yet to be added to `PayPalWebCheckout`. In fact, i would put it as a SPIKE ticket, because i dont know if we can navigate the web flow through an integration test.